docs/fsharp/tour.md

```diff
@@ -114,64 +114,51 @@ Additionally, DUs also support recursive definitions, allowing you to easily rep
+[Active Patterns](language-reference/active-patterns.md) are another powerful construct to use with pattern matching.  They allow you to partition input data into custom forms, decomposing them at the pattern match call site.  They can also be parameterized, thus allowing to define the partition as a function.  Expanding the previous example to support Active Patterns looks something like this:
 
 [!code-fsharp[ActivePatterns](~/samples/snippets/fsharp/tour.fs#L777-L795)]
 
-## Optional Types
+## Options
 
 One special case of Discriminated Union types is the Option Type, which is so useful that it's a part of the F# core library.
 
-[The Option Type](./language-reference/options.md) is a type that represents one of two cases: a value, or nothing at all.  It is used in any scenario where a value may or may not result from a particular operation.  This then forces you to account for both cases, making it a compile-time concern rather than a runtime concern.  These are often used in APIs where `null` is used to represent "nothing" instead, thus eliminating the need to worry about `NullReferenceException` in many circumstances.
+[The Option Type](language-reference/options.md) is a type that represents one of two cases: a value, or nothing at all.  It is used in any scenario where a value may or may not result from a particular operation.  This then forces you to account for both cases, making it a compile-time concern rather than a runtime concern.  These are often used in APIs where `null` is used to represent "nothing" instead, thus eliminating the need to worry about `NullReferenceException` in many circumstances.
 
 [!code-fsharp[Options](~/samples/snippets/fsharp/tour.fs#L803-L823)]
 
 ## Units of Measure
 
-One unique feature of F#'s type system is the ability to provide context for numeric literals through Units of Measure.
-
-[Units of Measure](./language-reference/units-of-measure.md) allow you to associate a numeric type to a unit, such as Meters, and have functions perform work on units rather than numeric literals.  This enables the compiler to verify that the types of numeric literals passed in make sense under a certain context, thus eliminating runtime errors associated with that kind of work.
+F#'s type system includes  the ability to provide context for numeric literals through [Units of Measure](language-reference/units-of-measure.md). These allow you to associate a numeric type to a unit, such as Meters, and have functions perform work on units rather than numeric literals.  This enables the compiler to verify that the types of numeric literals passed in make sense under a certain context, thus eliminating runtime errors associated with that kind of work.
```

Comment:
```suggestion
F#'s type system includes  the ability to provide context for numeric literals through [Units of Measure](language-reference/units-of-measure.md). Units of measure allow you to associate a numeric type to a unit, such as Meters, and have functions perform work on units rather than numeric literals.  This enables the compiler to verify that the types of numeric literals passed in make sense under a certain context, thus eliminating run-time errors associated with that kind of work.
```